### PR TITLE
Narrow Bill of Lading Organization schema

### PR DESCRIPTION
**Pull request by BenjaminMoe**
The scope of this pull request is that instead of pulling in all of the attributes from the Organization schema, it only pulls in the fields that are relevant to the bill of lading schema.

### CODE REVIEW

**Review by nissimsan — APPROVED** [review]

**Comment by nissimsan** [comment]
Relates to https://github.com/w3c-ccg/traceability-vocab/issues/624

File: docs/openapi/components/schemas/common/BillOfLading.yml [hunk 1]
```diff
@@ -59,7 +59,71 @@ properties:
...
+          - Organization
+        default:
+          - Organization
+        items:
+          type: string
+          enum:
+            - Organization
+      id:
+        title: Identifier
+        description: Decentralized identifier for the carrier
```
Comment (TallTed) [inline docs/openapi/components/schemas/common/BillOfLading.yml:77]:
```suggestion
        description: Decentralized identifier for the carrier.
```

File: docs/openapi/components/schemas/common/BillOfLading.yml [hunk 2]
```diff
@@ -69,15 +133,143 @@ properties:
...
+          - Organization
+        default:
+          - Organization
+        items:
+          type: string
+          enum:
+            - Organization
+      id:
+        title: Identifier
+        description: Decentralized identifier for the consignor
```
Comment (TallTed) [inline docs/openapi/components/schemas/common/BillOfLading.yml:151]:
```suggestion
        description: Decentralized identifier of the consignor.
```

**Review by brownoxford — APPROVED** [review]

**Comment by nissimsan** [comment]
@BenjaminMoe , pls merge once conflict is resolved

**Pull request merged**